tools/pipelines/templates/build-npm-package.yml

```diff
@@ -259,6 +259,12 @@ stages:
           # and Istanbul's cobertura reporter doesn't escape them causing parse error when we publish
           # A quick fix to patch the file with sed. (See https://github.com/bcoe/c8/issues/302)
           - ${{ if eq(variables['testCoverage'], true) }}:
+            - task: Npm@1
+              displayName: npm run ci:test:coverage
+              inputs:
+                command: 'custom'
+                workingDir: ${{ parameters.buildDirectory }}
+                customCommand: 'run ci:test:coverage'
```

Comment:
I'm not sure what this task is for. Does it cause the tests to run again?

Reply:
> I'm not sure what this task is for. Does it cause the tests to run again?

I have removed this part, it seems I kept that when I resolved the conflict with Curtis' latest update